README.md

```diff
@@ -33,10 +33,12 @@ Create a GitHub Source button:
 4. Modify the parameters below to create a uniform button:
     - *badgecontent*: Open on GitHub-black
     - *logo*: github
-    - *link*: Paste the copied GitHub repo URL
-   
-<img alt="Static Badge" src="https://img.shields.io/badge/Open%20on%20GitHub-black?style=flat&logo=github&link=https%3A%2F%2Fgithub.com%2Fdanforthcenter%2Fplantcv-tutorial-template">
-
+    - *link*: copy markdown code
```

Comment:
No need to do anything with the link field

Reply:
I removed the link option then if we don't need it.